crates/consensus/src/transaction/eip4844.rs

```diff
@@ -914,6 +954,7 @@ impl Transaction for TxEip4844WithSidecar {
 /// This represents a set of blobs, and its corresponding commitments and proofs.
 #[derive(Debug, Default, Clone, PartialEq, Eq, Hash)]
 #[repr(C)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Can we proactively dedupe this as part of this PR?

use crate::kzg::{Blob, Bytes48};
use serde::{Deserialize, Serialize};

current code in RPC types is as follows
```
/// This represents a set of blobs, and its corresponding commitments and proofs.
#[derive(Clone, Debug, PartialEq, Eq, Hash, Default, Serialize, Deserialize)]
#[repr(C)]
pub struct BlobTransactionSidecar {
    /// The blob data.
    pub blobs: Vec<Blob>,
    /// The blob commitments.
    pub commitments: Vec<Bytes48>,
    /// The blob proofs.
    pub proofs: Vec<Bytes48>,
}
```

Reply:
@prestwich alloy-ending-types relies on `Blob` and `Bytes48` having `ssz` support and c-kzg types does not have it, what's the desired way to handle this?

Reply:
manually impl ssz trait with a helper struct? If it's getting that complicated, it should go in a separate PR though